### ISSUE TITLE
2 diiferents screen

### ISSUE THREAD

**Issue by Piter00** [MindShow/USBDisplay#46]
How can 1 use as 2 different screen, like on of them as a monitor

**Comment by MindShow** [comment]
windows support extend mode

**Comment by Piter00** [comment]
I'm looking for an extended mode for MacBook

**Comment by Piter00** [comment]
can we drive the adaptateur to make it compatible with display link app ? for use extension to extend screen

**Comment by Piter00** [comment]
or create airplay option recognized

**Comment by ronaldotantra** [comment]
yes i need extended mode too, any solution ? on macos sonoma ? please mention me if there is thanks
